### DISCUSSION TITLE
The file has an unsupported compression type [Premiere Pro]

### DISCUSSION THREAD

**Discussion by adrianoalfaro** [mifi/lossless-cut#883] [Support]
Hello, thank you for this tool, it is so useful!

I use it to cut DJI Mini 2 4K footage, that I then edit with Premiere Pro 2021. Sometimes happens that I can't import cuts into the Premiere Pro timeline because of the error "The file has an unsupported compression type".
On a series of 8 MP4, 3 of them give the error in Premiere. I also tried to cut again those MP4, but they still won't import in Premiere.
May I send you those file for inspection?

Thank you so much!

**Comment by mifi** [comment]
Which cut mode are you using? Normal or Keyframe cut? Also are you exporting as MOV or MP4? can you try to toggle between these options? Another option to try is the export option "Shift timestamps (avoid_negative_ts)", see if it makes a difference. You could send me a file finstaden@gmail.com but I don't have Premiere Pro so I cannot do much testing. If you find that one of my suggestions helps, then it would be nice if you report back so we can learn from this. Thanks

**Reply by adrianoalfaro** [reply]
> Which cut mode are you using? Normal or Keyframe cut? Also are you exporting as MOV or MP4? can you try to toggle between these options? Another option to try is the export option "Shift timestamps (avoid_negative_ts)", see if it makes a difference. You could send me a file [finstaden@gmail.com](mailto:finstaden@gmail.com) but I don't have Premiere Pro so I cannot do much testing. If you find that one of my suggestions helps, then it would be nice if you report back so we can learn from this. Thanks

Hello! Normal or Keyframe cut doesn't make any difference. I was exporting as MP4 (same as source). Now I tried to export as MOV and Premiere succesfully imported the videos.
Just to let you know, I cut lots of other MP4 from the DJI Mini 2 (exported as MP4), then I imported without any kind of problems in Premiere Pro. I will send you a sample file, so you can check in what it differs from a working one. Thank you!

**Reply by mifi** [reply]
Thanks for sending over. Nice view btw;)
It seems like the difference between the two files is that they have slightly different 'type" `Normal` vs `QuickShot`:

```
working file:      comment: 'DE=None,SN=1SFOH9V0AB0BE7, Type=Normal, HQ=Normal, Mode=P',
not working file:  comment: 'DE=None,SN=1SFOH9V0AB0BE7, Type=QuickShot, HQ=Normal, Mode=P',
```

Also the non-working one has a subtitle track, while the working one does not:

```
{
  index: 2,
  codec_name: 'mov_text',
  codec_long_name: 'MOV text',
  codec_type: 'subtitle',
  codec_time_base: '0/1',
  codec_tag_string: 'text',
  codec_tag: '0x74786574',
  r_frame_rate: '0/0',
  avg_frame_rate: '0/0',
  time_base: '1/30000',
  start_pts: 0,
  start_time: '0.000000',
  duration_ts: 600000,
  duration: '20.000000',
  bit_rate: '2048',
  nb_frames: '20',
  disposition: {
    default: 1,
    dub: 0,
    original: 0,
    comment: 0,
    lyrics: 0,
    karaoke: 0,
    forced: 0,
    hearing_impaired: 0,
    visual_impaired: 0,
    clean_effects: 0,
    attached_pic: 0,
    timed_thumbnails: 0,
  },
  tags: {
    creation_time: '2021-10-03T10:34:35.000000Z',
    language: 'eng',
    handler_name: '\x10DJI.Subtitle',
  },
}
```

I'm guessing that the MOV text subtitle type only works properly with MOV files then, and not MP4.

**Reply by adrianoalfaro** [reply]
Hello mifi,
I would like to know if is there any way I can fix a broken video by changing "QuickShot" to "Normal" and by delating the subtitle track... How can I manually edit the file?
Thank you so much!

**Reply by mifi** [reply]
Hi. Unfortunately I don't know how to change from QuickShot to Normal, as it's something embedded into the codec. But you said that exporting to MOV works, no?

**Comment by adrianoalfaro** [comment]
Thank you for taking a look!
